app/views/training/pages/_citation.html.slim

```diff
@@ -1,8 +1,8 @@
-.hf-citation
-  .gem-c-metadata data-module="gem-toggle metadata" data-gem-toggle-module-started="true" data-metadata-module-started="true"
-    dl.gem-c-metadata__list data-module="gem-track-click" data-gem-track-click-module-started="true"
-      dt.gem-c-metadata__term Published:
-      dd.gem-c-metadata__definition = format_date(content.created_at.to_fs)
-      - if updated_content?
-        dt.gem-c-metadata__term Updated:
-        dd.gem-c-metadata__definition data-module="ga4-link-tracker" data-ga4-track-links-only="" data-ga4-link="{&quot;event_name&quot;:&quot;navigation&quot;,&quot;type&quot;:&quot;content history&quot;,&quot;section&quot;:&quot;Top&quot;,&quot;action&quot;:&quot;opened&quot;}" data-ga4-link-tracker-module-started="true" #{format_date(content.updated_at.to_fs)}
+.cdt-citation
```

Comment:
What does the prefix `cdt` do for us? Just because a frontend designer did it doesn't mean it serves a purpose for us. Is this just a decorative wrapper for all of the elements?

Reply:
I have updated this to `.cdt-wrapper`